coalib/bears/Bear.py

```diff
@@ -305,6 +310,45 @@ def log_message(self, log_message, timestamp=None, **kwargs):
     def run(self, *args, dependency_results=None, **kwargs):
         raise NotImplementedError
 
+    def setup_profile(self, prof=None):
+        path = os.getcwd()
+        if self.profile == 'True':
+            filename = '{}_{}.prof'.format(self.section.name, self.name)
+            open(filename, 'wb')
+            prof.dump_stats(filename)
+        else:
+            subdir = self.profile
```

Comment:
you should check if it's actually an absolute path. So
1. it's a relative path: append it to os.getcwd
2. it's an absolute path: ignore os.getcwd

Reply:
I guess the API saving the dumps already respect that, so in fact you don't need the `join` below, just pass `self.profile` to `prof.dump_stats` :)